doc/genmodel.py

```diff
@@ -463,18 +478,10 @@ def main():
         sys.exit(1)
     makedirs(TARGET_DIR, exist_ok=True)
 
-    if args.cpus == -1:
-        cpus = int(os.environ.get("SASMODELS_BUILD_CPUS", "0"))
-    else:
-        cpus = args.cpus
-    if cpus != 1 and not args.force:
-        import multiprocessing
-        p = multiprocessing.Pool(cpus if cpus > 0 else None)
-        rst_files = p.map(process_model, args.files)
-    else:
-        rst_files = [process_model(py_file, args.force)
+    print("** 'Normal' processing **")
```

Comment:
Has multiprocessing been used at all (for any platform)? I guess we can afford slower docs build for sake of stability (better than not building at all).

Reply:
It's got to be faster than failing after 6hrs! ;)